<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Simulator</class>
 <widget class="QMainWindow" name="Simulator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:48pt;&quot;&gt;无人超市&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Authentication">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>160</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>身份验证</string>
    </property>
   </widget>
   <widget class="QPushButton" name="productPurchase">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>商品选购</string>
    </property>
   </widget>
   <widget class="QPushButton" name="maskDetection">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>110</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>防疫检测</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deal">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>260</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>商品结算</string>
    </property>
   </widget>
   <widget class="QPushButton" name="purchaseRecord">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>310</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>购买记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="cargoManagement">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>360</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>进出货管理</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addUser">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>410</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>添加用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="count">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>450</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>热销商品</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
